### PR TITLE
XRDDEV-704

### PR DESCRIPTION
**Pull request by iluwatar**
- Update Security Server user guide with REST OpenAPI 3 type service and endpoint management

### CODE REVIEW

**Comment by petkivim** [comment]
This could be added to section 7 - just before section 7.1:

> It is possible to define access rights on two levels for REST services:
>
> - REST service level
> - endpoint level. 
>
> In general, a REST service usually has multiple endpoints. When access rights are defined on the service level, they apply to all the endpoints of the REST service. Instead, defining access rights on the endpoint level gives access to specific endpoint(s) only.

**Comment by petkivim** [comment]
X-Road user guides and manuals do not contain screenshots currently. To keep the documents consistent, screenshots should probably not be added for REST related sections either. Therefore, I'd remove the screenshots from the Security Server user guide.

**Comment by iluwatar** [comment]
Pushed the changes

File: doc/Manuals/ug-ss_x-road_6_security_server_user_guide.md [hunk 1]
```diff
@@ -983,6 +990,27 @@ To change service parameters, follow these steps.
 3.  In the window that opens, configure the service parameters. To apply the selected parameter to all services described in the same service description, select the checkbox adjacent to this parameter in the **Apply to All in WSDL** column. To apply the configured parameters, click **OK**.
 
 
+### 6.7 Managing REST Endpoints
+
+**Access rights:** [Service Administrator](#xroad-service-administrator)
+
+REST type service descriptions can contain API endpoints. The purpose of the endpoints is more fine-grained access control. More about that in chapter [7 Access Rights](#7-access-rights).
+
+When URL type of the REST service is an OpenAPI 3 description, endpoints are parsed from the service description automatically. Additionally manual endpoints can be added as needed. When URL type is REST API base path, all the endpoints need to be created manually.
```
Comment (petkivim) [inline doc/Manuals/ug-ss_x-road_6_security_server_user_guide.md]: When URL type of the REST service is an OpenAPI 3 description, endpoints are parsed from the service description automatically. These endpoints cannot be manually updated or deleted. Additionally manual endpoints can be added as needed. When URL type is REST API base path, all the endpoints need to be created manually. Manually created endpoints can also be edited and deleted as needed.

Comment (iluwatar) [reply doc/Manuals/ug-ss_x-road_6_security_server_user_guide.md]: Ok

File: doc/Manuals/ug-ss_x-road_6_security_server_user_guide.md [hunk 2]
```diff
@@ -63,6 +63,7 @@ Doc. ID: UG-SS
  01.07.2019 | 2.28    | Changing the Security Server Owner chapter added (Chapter [3.4](#34-changing-the-security-server-owner)) | Petteri Kivimäki
  14.08.2019 | 2.29    | Added automatic backups | Ilkka Seppälä
  30.09.2019 | 2.30    | Added remote database migration guide | Ilkka Seppälä
+ 14.09.2019 | 2.31    | Updated REST services in chapter [6] | Ilkka Seppälä
```
Comment (petkivim) [inline doc/Manuals/ug-ss_x-road_6_security_server_user_guide.md]: This should probably be 14.10.2019.

Comment (iluwatar) [reply doc/Manuals/ug-ss_x-road_6_security_server_user_guide.md]: Ok

**Pull request merged**